report/4.Design/HardwareTest.tex

```diff
@@ -0,0 +1,24 @@
+\section{Hardware testing}
+To ensure the most accurate of the device, it is important to ensure the quality and precision of the chosen hardware to be at its highest.
```

Comment:
"To ensure the most accurate of the device"

"To ensure the most accuracy of the device"